RRTstar:
  map:
    OBSTACLE_THRESHOLD: 50    # 地图中栅格代价值大于等于(>=)该值的栅格，会被视为障碍物，搜索过程中将直接跳过该栅格

  sample:
    ITERATOR_TIMES: 100000    # 最大迭代次数
    GOAL_SAMPLE_RATE: 0.1     # 终点采样率，即每走一步，有多少的概率随机选取终点作为新的终点
    GOAL_DIS_TOLERANCE: 2.0   # 新增长节点若与终点的距离在该范围内，则直接将终点作为新增长节点
    STEP_SIZE: 3.0            # 每次扩展节点的步长
    NEAR_DIS: 10.0            # 在扩展完新节点后，再该范围内重新璇姐父节点、重新排线